packages/apps/vpc/values.yaml

```diff
@@ -0,0 +1,16 @@
+##
+## @section Common parameters
+##
+
+## @typedef {struct} Subnet - Subnet of a VPC
+## @field {Subnet} [name] - Subnet name tag
```

Comment:
Huh?
You're defining a struct like
```go
type Subent struct {
    name Subnet
    cidr string
}
```
This looks quite wrong to me and leads to the strange schema generated up [here](https://github.com/cozystack/cozystack/pull/1543/files#r2481083404)